webapp/src/main/java/com/box/l10n/mojito/service/tm/TMTextUnitVariantRepository.java

```diff
@@ -85,46 +86,46 @@ Page<TextUnitVariantDeltaDTO> findAllUsedForRepositoryAndLocalesInDateRange(
    */
   @Query(
       nativeQuery = true,
-      value =
-          "select distinct base_tu.name as textUnitName, "
-              + "   l.bcp47_tag as bcp47Tag, "
-              + "   latest_tuv.content as content, "
-              + "   case "
-              + "      when previous_tuv.id is null then 'NEW_TRANSLATION' "
-              + "      else 'UPDATED_TRANSLATION' "
-              + "   end as deltaType "
-              + "from tm_text_unit_variant latest_tuv "
-              + "inner join tm_text_unit_current_variant base_tucv on base_tucv.tm_text_unit_variant_id = latest_tuv.id "
-              + "inner join tm_text_unit base_tu on base_tu.id = base_tucv.tm_text_unit_id "
-              + "inner join push_run_asset_tm_text_unit prattu on prattu.tm_text_unit_id = base_tu.id "
-              + "inner join push_run_asset pra on pra.id = prattu.push_run_asset_id "
-              + "inner join push_run pr on pr.id = pra.push_run_id and pr.id in :pushRunIds "
-              + "inner join asset a on a.id = base_tu.asset_id "
-              + "inner join repository r on r.id = a.repository_id and r.id = :repositoryId "
-              + "inner join locale l on l.id = latest_tuv.locale_id and l.id in :localeIds "
-              + "left outer join ( "
-              + "   select distinct previous_tuv.id, "
-              + "                   previous_tuv.content_md5, "
-              + "                   a.id as asset_id, "
-              + "                   l.id as locale_id, "
-              + "                   tu.id as text_unit_id "
-              + "   from tm_text_unit_variant previous_tuv "
-              + "       inner join pull_run_text_unit_variant prtuv on prtuv.tm_text_unit_variant_id = previous_tuv.id "
-              + "       inner join pull_run_asset pra on pra.id = prtuv.pull_run_asset_id "
-              + "       inner join pull_run pr on pr.id = pra.pull_run_id and pr.id in :pullRunIds "
-              + "       inner join tm_text_unit tu on tu.id = previous_tuv.tm_text_unit_id "
-              + "       inner join asset a on a.id = tu.asset_id "
-              + "       inner join repository r on r.id = a.repository_id and r.id = :repositoryId "
-              + "       inner join locale l on l.id = previous_tuv.locale_id and l.id in :localeIds "
-              + " ) as previous_tuv on previous_tuv.asset_id = a.id "
-              + "                   and previous_tuv.locale_id = l.id "
-              + "                   and previous_tuv.text_unit_id = base_tu.id "
-              + "where "
-              + "   latest_tuv.included_in_localized_file = true "
-              + // Exclude rejected translations
```

Comment:
@garionpin loosing the comments inside, only one case. I'll keep it stripped.

Reply:
@aurambaj - using SQL comments at the end of the lines here would probably work Ok for any future needs